projects/step-frontend/src/lib/modules/execution/components/execution-tree/execution-tree.component.ts

```diff
@@ -33,9 +34,9 @@ export class ExecutionTreeComponent implements TreeActionsService {
   readonly selectedNode$: Observable<ReportTreeNode | undefined> = this._treeState.selectedNode$;
 
   constructor(
-    @Inject(EXECUTION_TREE_PAGING) private _paging: ExecutionTreePaging,
-    private _treeState: TreeStateService<ReportNode, ReportTreeNode>,
-    private _treeUtils: ReportTreeNodeUtilsService
+    @Inject(EXECUTION_TREE_PAGING) private _paging: ExecutionTreePagingSetting,
+    private _executionTreePaging: ExecutionTreePaging,
+    private _treeState: TreeStateService<ReportNode, ReportTreeNode>
   ) {}
```

Comment:
Make sense to replace the constructor with fields, initialized by `inject` function

Reply:
As this is a fix for a minor release I want to touch least amount of code as possible